singlevenue/src/main/res/layout/activity_location.xml

```diff
@@ -50,12 +38,29 @@
                 android:layout_below="@+id/map_view_part"
                 android:layout_margin="3dp"
                 android:textSize="16dp" />
+
+            <TextView
+                android:id="@+id/location_loading_textview"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_below="@id/location_logo"
+                android:layout_marginBottom="50dp"
+                android:height="300sp"
+                android:background="@color/colorGrey"
+                android:gravity="center"
+                android:text="Loading..."
+                android:textColor="@color/colorWhite"
```

Comment:
Tiny nitpick, If the text colour is white, I think it might be hard to read on a light grey background.